### DISCUSSION TITLE
Plots very slow

### DISCUSSION THREAD

**Discussion by Kingson1874** [Chia-Network/chia-blockchain#6326] [Support]
Pls help me ，

Why has the speed of plots and migration slowed after version 1.1.6?

Before version 1.1.5, plots only took 9 hours, now it takes 15 hours.

**Comment by XIII39** [comment]
I'm not sure the software update would cause the plotting to slow.
Check the temperature of you SSD drives. Chances are they are running hot.. Hot drives = slow drives

**Comment by BrandtH22** [comment]
@Kingson1874 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
